drivers/SmartThings/matter-switch/src/test/test_matter_switch_device_types.lua

```diff
@@ -394,6 +460,77 @@ local function test_init_parent_child_unsupported_device_type()
   })
 end
 
```

Comment:
this isn't in a function? What are you trying to do here out of curiosity?

Reply:
ignore please, and respond to the one that notes the lines I'm asking about :sweat_smile: